### DISCUSSION TITLE
Textarea that grows with the text

### DISCUSSION THREAD

**Discussion by xl0** [saadeghi/daisyui#3182] [Ideas / feature requests]
Hi. This is a common use case that is a bit fiddly to implement correctly.

It would be awesome if there were a way to let the textarea grow/shrink with the number of lines, with configurable min/max lines.

**Comment by saadeghi** [comment]
There's a CSS attribute for that: `field-sizing:content` and you can use it as a Tailwind CSS class like `[field-sizing:content]`  
Example:
https://play.tailwindcss.com/LC0ZmhLDIz

You can also add min and max sizes: `min-h-[10rem] max-h-[20rem]`  
https://play.tailwindcss.com/7L3QLpt7ND

**Comment by xl0** [comment]
@saadeghi thank you!
